tests/integration/test_server.py

```diff
@@ -3,14 +3,17 @@
 import pytest
 
 from dask_sql.server.app import _init_app, app
+from dask_sql import Context
 
 # needed for the testclient
 pytest.importorskip("requests")
 
 
 @pytest.fixture(scope="module")
 def app_client():
-    _init_app(app)
+    c = Context()
+    c.sql("SELECT 1 + 1").compute()
+    _init_app(app, c)
```

Comment:
Any reason why we're manually creating a context for the standard server tests?

Reply:
The test fails if you run `pytest tests/integration/test_server.py` in isolation. And as I used this module as starter for test_jdbc.py I discovered after some messing about that tests/integration/test_compatibility.py  has to run first to ensure test_server.py and test_jdbc.py could run. It means both will now run independently.

Reply:
That makes sense, thanks for the explanation